lib/Wikia/src/Consul/Client.php

```diff
@@ -0,0 +1,90 @@
+<?php
+
+/**
+ * A helper class that uses sensiolabs/consul-php-sdk for getting the list of service nodes
+ *
+ * @see PLATFORM-1489
+ * @see https://www.consul.io/docs/agent/http/health.html#health_service
+ */
+
+namespace Wikia\Consul;
+
+use SensioLabs\Consul\ServiceFactory;
+use Wikia\Logger\WikiaLogger;
+
+class Client {
+
+	use \Wikia\Logger\Loggable;
+
+	protected $logger;
+
+	/* @var \SensioLabs\Consul\Services\Health $api */
+	protected $api;
+
+	function __construct() {
+		$this->logger = WikiaLogger::instance();
+
+		$consulService = new ServiceFactory( [], $this->logger );
+		$this->api = $consulService->get( 'health' );
+	}
+
+	/**
+	 * Returns IP addresses of given service healthy nodes
+	 *
+	 * $catalog->getNodes( 'db-a', 'slave' )
+	 *
+	 * $ curl "http://127.0.0.1:8500/v1/health/service/db-g?tag=slave&passing"
+	 *
+	 * @param string $service
+	 * @param string $tag
+	 * @return array list of IP addresses
+	 */
+	function getNodes( $service, $tag ) {
+		global $wgWikiaDatacenter;
+		$resp = $this->api->service( $service, [ 'tag' => $tag, 'dc' => $wgWikiaDatacenter, 'passing' => true ] )->json();
```

Comment:
:+1: 
although I think the DC  should be optional if not specified.
Consul by default will only fetch data from current datacenter the Agent is configured to be in.

But sometimes you might want to address resources from other DC for testing etc.